### PR TITLE
Flask secure cookies checks

### PR DESCRIPTION
**Pull request by minusworld**
Introduce checks for
1) secure session cookies in flask, set view app.config.update(...)
2) secure UESR cookies in flask (different from session cookies), set using response.set_cookie(...). This is an sgrep version of https://checks.bento.dev/en/latest/flake8-flask/secure-set-cookie/

### CODE REVIEW

File: python/flask/secure-cookies/session-secure-cookies-dict-set.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import flask
+
+# ok
+app = flask.Flask()
+
+app.config["SESSION_COOKIE_SECURE"]=True
+app.config["SESSION_COOKIE_HTTPONLY"]=True
+app.config["SESSION_COOKIE_SAMESITE"]='Lax'
```
Comment (ulziibay) [inline python/flask/secure-cookies/session-secure-cookies-dict-set.py]: Can you add more tests with different ordering of the config setting?

File: python/flask/secure-cookies/secure-cookies.yml [hunk 2]
```diff
@@ -2,19 +2,73 @@
...
-            ...,
-            SESSION_COOKIE_SECURE=True,
-            SESSION_COOKIE_HTTPONLY=True,
-            SESSION_COOKIE_SAMESITE="...",
-            ...
-          )
-      - pattern: |
-          $APP = flask.Flask(...)
-          ...
+      - pattern-either:
```
Comment (ulziibay) [inline python/flask/secure-cookies/secure-cookies.yml]: Can the rule also detect the configuration update in one function call like in https://flask.palletsprojects.com/en/1.1.x/security/#security-cookie?

File: python/flask/secure-cookies/secure-cookies.yml [hunk 3]
```diff
@@ -2,19 +2,73 @@
...
+                ...
+                $APP.config["SESSION_COOKIE_SAMESITE"] = $C
+                ...
+                $APP.config["SESSION_COOKIE_HTTPONLY"] = $B
+                ...
+                $APP.config["SESSION_COOKIE_SECURE"] = $A
+            - pattern: |
+                $APP = flask.Flask(...)
+                ...
     message: "Session cookies should be protected with the Secure, HttpOnly, and SameSite attributes. Set these attributes on your session cookies by setting SESSION_COOKIE_SECURE=True, SESSION_COOKIE_HTTPONLY=True, and SESSION_COOKIE_SAMESITE='Lax|Strict' in your application config."
```
Comment (ulziibay) [inline python/flask/secure-cookies/secure-cookies.yml]: We can link to https://flask.palletsprojects.com/en/1.1.x/security/#security-cookie

File: python/flask/secure-cookies/secure-cookies.yml (hunk 2 above)
Comment (minusworld) [reply python/flask/secure-cookies/secure-cookies.yml]: Yes, check out this test: https://github.com/returntocorp/sgrep-rules/blob/flask/secure-cookies/python/flask/secure-cookies/session-secure-cookies-set.py

File: python/flask/secure-cookies/session-secure-cookies-dict-set.py (hunk 1 above)
Comment (minusworld) [reply python/flask/secure-cookies/session-secure-cookies-dict-set.py]: Good idea!

**Pull request merged**